apps/web/lib/emails/templates/attendee-scheduled-email.ts

```diff
@@ -310,12 +310,16 @@ ${getRichDescription(this.calEvent)}
 
   protected getLocation(): string {
     let providerName = this.calEvent.location ? getAppName(this.calEvent.location) : "";
-
     if (this.calEvent.location && this.calEvent.location.includes("integrations:")) {
       const location = this.calEvent.location.split(":")[1];
       providerName = location[0].toUpperCase() + location.slice(1);
     }
 
+    // If location its a url, probably we should be validating it with a custom library
+    if (this.calEvent.location && /^https?:\/\//.test(this.calEvent.location)) {
```

Comment:
We can start doing a small lib that only valid http or https on @calcom/lib?
export function validHttp(url: string){}
And also using it below
apps/web/lib/emails/templates/organizer-scheduled-email.ts

Reply:
This was one of the random commits from main. It is out of scope of this PR.